Pcap++/src/RawSocketDevice.cpp

```diff
@@ -456,7 +456,7 @@ namespace pcpp
 		int fd = socket(AF_PACKET, SOCK_RAW, htobe16(ETH_P_ALL));
 		if (fd < 0)
 		{
-			PCPP_LOG_ERROR("Failed to create raw socket. Error code was " << errno);
+			PCPP_LOG_ERROR("Failed to create raw socket. Error code was " << strerror(errno));
```

Comment:
Not checked the full code so maybe not important but `strerror` is not thread safe

Reply:
Actually errno is thread since it is thread local but there are other strerror functions in library so better to fix it all later

Reply:
yes, I think you're right. I'll open a separate PR for it